Makefile

```diff
@@ -52,8 +54,8 @@ install::
 	mkdir -p "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)"
 	cp -r ${PACKDIR}/nodejs/bin/. "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)"
 	rm -rf "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)/node_modules"
+	cp -r ${PACKDIR}/nodejs/node_modules "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)"
```

Comment:
after blowing away the previous node modules, we copy over the existing ones we installed/linked for this module when building.  

This is necessary, otherwise we break transitive updates.  i.e. consider hte case of pulumi/pulumi, pulumi/aws, and pulumi/cloud.   With the previous logic, even if we linked ```@pulumi/pulumi``` into ```@pulumi/aws```, once we build ```@pulumi/aws```, we'd toss its node_modules, and then do a yarn install, giving us the modules it was writtne against, not what we'd linked locally.

If we then linked @pulumi/pulumi and @pulumi/aws into @pulumi/cloud, we'd be able to observe different @pulumi/pulumi libs.